packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -456,6 +457,11 @@ class FormLayout extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))
    * @protected
    */
   _updateLayout() {
+    // Do not update layout when invisible
+    if (isElementHidden(this)) {
+      return false;
```

Comment:
```suggestion
      return;
```
Should it just return undefined?

Reply:
Yes, my bad. I copied it from the method where it had a return value 🙈